app/src/pages/prompts/PromptsTable.tsx

```diff
@@ -0,0 +1,185 @@
+        if (
+          scrollHeight - scrollTop - clientHeight < 300 &&
+          !isLoadingNext &&
+          hasNext
+        ) {
+          loadNext(PAGE_SIZE);
+        }
+      }
+    },
+    [hasNext, isLoadingNext, loadNext]
+  );
+  const table = useReactTable({
+    columns: [
+      {
+        header: "name",
+        accessorKey: "name",
+        cell: ({ row }) => {
+          return <Link to={`${row.original.id}`}>{row.original.name}</Link>;
+        },
+      },
+      {
+        header: "created at",
+        accessorKey: "createdAt",
+        cell: TimestampCell,
+      },
+    ],
+    data: tableData,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    manualSorting: true,
+  });
+
+  const rows = table.getRowModel().rows;
+  const isEmpty = rows.length === 0;
+  return (
+    <div
+      css={css`
+        flex: 1 1 auto;
+        overflow: auto;
+      `}
+      onScroll={(e) => fetchMoreOnBottomReached(e.target as HTMLDivElement)}
+      ref={tableContainerRef}
+    >
+      <table css={selectableTableCSS}>
+        <thead>
+          {table.getHeaderGroups().map((headerGroup) => (
+            <tr key={headerGroup.id}>
+              {headerGroup.headers.map((header) => (
+                <th colSpan={header.colSpan} key={header.id}>
+                  {header.isPlaceholder ? null : (
+                    <div
+                      {...{
+                        className: header.column.getCanSort()
+                          ? "cursor-pointer"
+                          : "",
+                        onClick: header.column.getToggleSortingHandler(),
+                        style: {
+                          textAlign: header.column.columnDef.meta?.textAlign,
+                        },
+                      }}
+                    >
+                      {flexRender(
+                        header.column.columnDef.header,
+                        header.getContext()
+                      )}
+                      {header.column.getIsSorted() ? (
+                        <Icon
+                          className="sort-icon"
+                          svg={
+                            header.column.getIsSorted() === "asc" ? (
+                              <Icons.ArrowUpFilled />
+                            ) : (
+                              <Icons.ArrowDownFilled />
+                            )
+                          }
+                        />
+                      ) : null}
+                    </div>
+                  )}
+                </th>
+              ))}
+            </tr>
+          ))}
+        </thead>
+        {isEmpty ? (
+          <TableEmpty />
+        ) : (
+          <tbody>
+            {rows.map((row) => {
+              return (
+                <tr
+                  key={row.id}
+                  onClick={() => {
+                    navigate(`${row.original.id}`);
+                  }}
+                >
```

Comment:
This one I have a rationale for - it's that I at least place an anchor tag on the name so that becomes the main "accessible" way to navigate. The click on the TR is just meant to solve the "I just want to click on the full row" but maintains the DOM structure needed.

Reply:
Gotcha, sounds good